app/bundles/EmailBundle/Model/EmailModel.php

```diff
@@ -2128,6 +2128,38 @@ public function getLookupResults($type, $filter = '', $limit = 10, $start = 0, $
         return $results;
     }
 
+    /**
+     * @param string              $type
+     * @param string              $filter
+     * @param int                 $limit
+     * @param int                 $start
+     * @param array<string,mixed> $options
+     *
+     * @return array<string,array<string,string>>
+     */
+    public function getLookupResultsWithIdName(
+        $type, $filter = '', $limit = 10, $start = 0, $options = [],
+    ): array {
```

Comment:
On the second look, could we use types directly?
```suggestion
    /**
     * @param array<string,mixed> $options
     *
     * @return array<string,array<string,string>>
     */
    public function getLookupResultsWithIdName(
        string $type, string $filter = '', int $limit = 10, int $start = 0, array $options = [],
    ): array {
```